src/components/interfaces/Header/Notifications.tsx

```diff
@@ -22,9 +22,10 @@ export const Notifications = () => {
     <Sheet>
       <SheetTrigger asChild>
         <Button
+          size={null}
           variant="ghost"
           aria-label="Notifications"
-          className="size-8! p-0!"
+          className="size-8 rounded-md p-0"
```

Comment:
When there is no `size`, then `p-0` would also be unnecessary, as padding is applied with size. 

```suggestion
          className="size-8 rounded-md"
```